### ISSUE TITLE
Ghosts can see who antags are by examining their mood

### ISSUE THREAD

**Issue by PKPenguin321** [tgstation/tgstation#42202]
Antagonists get the antagonist mood buff, which ghosts can apparently see by examining them. Pretty sure this is unintended as we deliberately chose not to give ghosts antaghud.

**Comment by Tlaltecuhtli** [comment]
feature

**Comment by Tlaltecuhtli** [comment]
else are you gonna mute dead people from dchat + disable observing command + disable syndi chat + make ghosts unable to see ai laws?

**Comment by YakumoChen** [comment]
Or just disable ghosts viewing moodies

**Comment by Vidiveni** [comment]
Aw come on, I want this bug to stay. It's a massive quality of life 'feature'. 

>Get killed
>Check moodlet
Okay it was an antag, then all is fine. 

vs

>Get killed
>Ask in deadchat if anyone knows if that person is antag
>Ahelp after round end because you're surprised it wasn't an antag
>Get angry bwoink by admin ahelping after round end

Also useful for observers looking for antags to observe.

**Comment by Cyprex** [comment]
Checking if someone is antag is trivial, so ahelp anyway.

**Comment by Qustinnus** [comment]
this isn't true and whoever claimed this is a thing, is an idiot. can this be closed as its not an actual thing @SpaceManiac . antag moodlets dont get used for examine calculations.

**Comment by Qustinnus** [comment]
```		
                mood += event.mood_change
		if(!event.hidden)
			shown_mood += event.mood_change```

```
	GET_COMPONENT_FROM(mood, /datum/component/mood, src)
	if(mood)
		switch(mood.shown_mood)
			if(-INFINITY to MOOD_LEVEL_SAD4)
				msg += "[t_He] look[p_s()] depressed.\n"
			if(MOOD_LEVEL_SAD4 to MOOD_LEVEL_SAD3)
				msg += "[t_He] look[p_s()] very sad.\n"
			if(MOOD_LEVEL_SAD3 to MOOD_LEVEL_SAD2)
				msg += "[t_He] look[p_s()] a bit down.\n"
			if(MOOD_LEVEL_HAPPY2 to MOOD_LEVEL_HAPPY3)
				msg += "[t_He] look[p_s()] quite happy.\n"
			if(MOOD_LEVEL_HAPPY3 to MOOD_LEVEL_HAPPY4)
				msg += "[t_He] look[p_s()] very happy.\n"
			if(MOOD_LEVEL_HAPPY4 to INFINITY)
				msg += "[t_He] look[p_s()] ecstatic.\n"
	msg += "*---------*</span>"
```

**Comment by imsxz** [comment]
how is this a bug? this is 100% intentional behavior afaik.

**Comment by SpaceManiac** [comment]
closing on Qust's word as not real

**Comment by AlexTheSysop42** [comment]
observe --> look at moodlet

**Comment by Qustinnus** [comment]
that shows hud? thats gay

**Comment by AlexTheSysop42** [comment]
it shows everything they see afaik, even different hud themes (clockwork, etc)

**Comment by Qustinnus** [comment]
ughhhhhhhhhhhhhhhhhhhhhhhhhhh okay then thats a bug

**Comment by SpaceManiac** [comment]
I mean if they're like a ling or something you'll be able to see their action buttons too, so

**Comment by AlexTheSysop42** [comment]
you cant see action buttons for some reason, although you were able to see the "hide buttons" button (but that may have been changed)

**Comment by necromanceranne** [comment]
I HIGHLY doubt people would be as brazen as to use it to metagrudge, and if that were the case it would be no different to using ghost roles to do the same.

At the moment it saves having to ahelp.

**Comment by Cyprex** [comment]
Again, I do not believe "it saves having to ahelp" is a good reason here, due to this being trivial to do for admins. That said, I do not believe it is feasible or sane to remove this feature.

**Comment by improvedname** [comment]
If people wanted to metagrudge they wouldnt give a shit about the moodlet since at the they kill you they are valid

**Comment by wesoda25** [comment]
It does help save an ahelp

**Comment by k-k-karma** [comment]
This is not an exploit. You can easily determine if someone's antag by BEING A FUCKING GHOST IN THE FIRST PLACE

**Comment by nicbn** [comment]
Yes it is an exploit, we chose not to show antag hud if you're a ghost.
The best solution may be to hide target hud when observing.

**Comment by k-k-karma** [comment]
no it isnt lmao
figuring out if someone's antag is already very easy, look in their bags as ghost

**Comment by nicbn** [comment]
> we chose not to show antag hud if you're a ghost.

> The best solution may be to hide target hud when observing.

**Comment by coiax** [comment]
This will be partially fixed by #42157, but not completely, since you can scan people for the absence of the SAW SOMEONE DIE mood, not a hard tell though

**Comment by k-k-karma** [comment]
> > we chose not to show antag hud if you're a ghost.
> 
> > The best solution may be to hide target hud when observing.

their hud isn't a fucking backpack is it you fucking moron
